eppoclient/configurationrequestor.go

```diff
@@ -36,24 +40,21 @@ func (ecr *experimentConfigurationRequestor) GetConfiguration(experimentKey stri
 }
 
 func (ecr *experimentConfigurationRequestor) FetchAndStoreConfigurations() {
-	var responseBody map[string]json.RawMessage
-
-	configs := dictionary{}
 	result := ecr.httpClient.get(RAC_ENDPOINT)
+	var wrapper racResponse
 
-	err := json.Unmarshal([]byte(result), &responseBody)
-
+	// Unmarshal JSON data directly into the wrapper struct
+	err := json.Unmarshal([]byte(result), &wrapper)
 	if err != nil {
-		fmt.Println("Failed to unmarshal RAC response json", result)
+		fmt.Println("Failed to unmarshal RAC response JSON", result)
 		fmt.Println(err)
+		return
 	}
 
-	err = json.Unmarshal(responseBody["flags"], &configs)
-
+	// Now wrapper.Flags contains all configurations mapped by their keys
+	// Pass this map directly to SetConfigurations
+	err = ecr.configStore.SetConfigurations(wrapper.Flags)
```

Comment:
We can remove a `json.Unmarshal` because it has already happen when creating the `racResponse` object. Now can just reference `Flags`